pkgs/applications/networking/instant-messengers/mikutter/default.nix

```diff
@@ -54,7 +69,6 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    broken = true;
```

Comment:
Should not this still be marked as broken when you cannot log in?

Reply:
The build itself is not broken, and there are no tests for this... For any bugs during usage we can open new issues but it seems like this has no users yet.